src/components/Lesson/ImprovePage.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../styles/variables';
+
+.link {
+  text-decoration: none !important;
+  text-align: center;
+  font-size: 16px;
```

Comment:
Anbefaler forøvrig å google litt rundt dersom du er interessert i denne diskusjonen (pass på at du leser artikler som ikke er eldre enn et par år, siden disse tingene utvikler seg fort).